src/jit_channel/msgs.rs

```diff
@@ -0,0 +1,262 @@
+use std::convert::TryFrom;
+
+use bitcoin::hashes::hmac::{Hmac, HmacEngine};
+use bitcoin::hashes::sha256::Hash as Sha256;
+use bitcoin::hashes::{Hash, HashEngine};
+use serde::{Deserialize, Serialize};
+
+use crate::transport::msgs::{LSPSMessage, RequestId, ResponseError};
+use crate::utils;
+
+pub(crate) const LSPS2_GETVERSIONS_METHOD_NAME: &str = "lsps2.getversions";
+pub(crate) const LSPS2_GETINFO_METHOD_NAME: &str = "lsps2.getinfo";
```

Comment:
Spec changed to `get_info`.
```suggestion
pub(crate) const LSPS2_GETINFO_METHOD_NAME: &str = "lsps2.get_info";
```